src/status_im/ui/screens/desktop/main/tabs/home/views.cljs

```diff
@@ -62,9 +63,26 @@
        [react/view {:style styles/timestamp}
         [chat-item/message-timestamp (:timestamp last-message)]]])))
 
-(defn chat-list-item [[chat-id chat]]
-  [react/touchable-highlight {:on-press #(re-frame/dispatch [:chat.ui/navigate-to-chat chat-id])}
-   [chat-list-item-inner-view (assoc chat :chat-id chat-id)]])
+(defn chat-list-item [[chat-id
+                       {:keys [group-chat public? public-key] :as chat}]]
+  [popup-menu/wrap-with-menu
+   [chat-list-item-inner-view (assoc chat :chat-id chat-id)]
+   (remove nil?
```

Comment:
which set of chats are expected to not be removed here? just private, or are we up to additional categories? :eye:

Reply:
`remove` affects menu items for this particular chat item, not the chat list. So for public/group chats we cannot execute `View Profile`, for example.  Will write the doc for `wrap-with-menu`, sorry for confusion.

Reply:
ah, no that makes total sense. my own confusion for missing at what level the `remove` was acting upon. thanks!